lib/auth/auth.ts

```diff
@@ -161,6 +161,20 @@ function doAuth(
     return cb(errors.InternalError);
 }
 
+/**
+ * This function will generate a version 4 content-md5 header
+ * It looks at the request path to determine what kind of header encoding is required
+ *
+ * @param path - the request path
+ * @param payload - the request payload to hash
+ */
+function generateContentMD5Header(
+    path: string,
+    payload: string,
+) {
+    const encoding = path && path.includes('backbeat') ? 'hex' : 'base64';
```

Comment:
The check looks too broad to me, although unlikely any path may contain `backbeat` without being targeted at a backbeat route. Would something like `path.startsWith('/_/backbeat')` work instead?

Reply:
Yep, sounds good!